### DISCUSSION TITLE
License for data?

### DISCUSSION THREAD

**Discussion by brainwo** [fdw/rofimoji#152] [Q&A] [answered]
Hi, I'm wondering if [data](https://github.com/fdw/rofimoji/tree/main/src/picker/data) has any license, or does it follow the repository license?
I would like to use the data on my projects, mainly the CJK ones.

Also, where do these datasets come from? Are there any source of it?

Thanks in advance.

**Comment by fdw** [comment, marked as answer]
That's a really good question about the licenses for data... I would say they are also MIT-licensed, but I have to check with the original data and its license. Do you have a suggestion where I could put this information?

In general, the data is scraped from somewhere, using the extractors in [`extractors`]. There you can also see where the data is originally taken from. In the case of CJK (and all other Unicode files), it's directly from the Unicode Consortium, covered by its own Terms of Service and (partly) by the "Data Files and Software License".

**Reply by brainwo** [reply]
I'm not that good on this matter. But if you asked me:
- I would first check the README, then
- see if there is an additional LICENSE file on the project root, or
- check the list of package contents from my system package manager.

Since the data is CSV I don't think it's a good idea to put the license as a comment on top of the file. It could help much to be able to process the data with a simple data processor.
